x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form.tsx

```diff
@@ -365,6 +365,22 @@ export const EditTransformFlyoutForm: FC<EditTransformFlyoutFormProps> = ({
               }
             )}
           />
+          <EditTransformFlyoutFormTextInput
+            dataTestSubj="transformEditFlyoutNumFailureRetriesInput"
+            errorMessages={formFields.numFailureRetries.errorMessages}
+            helpText={i18n.translate(
+              'xpack.transform.transformList.editFlyoutFormNumFailureRetriesHelpText',
+              {
+                defaultMessage:
+                  'Defines number of retries on a recoverable failure before the transform task is marked as `failed`. If not set, the cluster setting `xpack.transform.num_transform_failure_retries` or a default of 10 will be used.',
```

Comment:
I don't think the quotes are needed around `failed`.

Reply:
I think this should mention that `-1` means unlimited.

As it stands `-1` is only explained in the error message.

Reply:
Updated here https://github.com/elastic/kibana/pull/135486/commits/8dfffbcce7a6ecbfb7e9635b83ad355027bb0794